### PR TITLE
Improve ug and fix error messages

### LINKED ISSUE #123: [PE-D][Tester D] UI: Long strings on small placeholders
Long strings in small placeholders such as email and phone results in the data being truncated and inaccessible

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: dwangwk/ped#1

### LINKED ISSUE #145: [PE-D][Tester B] Unexpected behaviour
When trying to update the telegram handle, the error message states that it cannot be blank.

However, when I entered a blank input: `uc 1 t/`, 

The command passed successfully.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: agreatdayy/ped#5

### PR DESCRIPTION
**Pull request by kailashgautham**
Closes #123 
Closes #145

### CODE REVIEW

File: src/main/resources/view/PersonListCard.fxml
Comment (hiivan) [inline src/main/resources/view/PersonListCard.fxml:1]: May I ask the reason for this change? Just want to understand how removing FlowPane and implementing TextFlow helps.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ The following parameters are used for commands:
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `ac n/NAME`, `NAME` is a parameter which can be used as `ac n/John Doe`.
 
-* Items in square brackets are optional.<br>
+* Items in square brackets are optional. If an optional prefix's value is left blank, it is assumed that you are leaving the value blank or deleting the already existing value.<br>
```
Comment (g-tejas) [inline docs/UserGuide.md:217]: Feels repeated? What about "If an optional prefix's value is left blank, it is assumed that are deleting the already existing value."

**Review by g-tejas — CHANGES_REQUESTED** [review]
LGTM! And curious about what Ivan asked also

File: docs/UserGuide.md (hunk 1 above)
Comment (teojunda) [reply docs/UserGuide.md:217]: I think the current sentence is good, since for the add command, inputing `t/` will leave the value blank.

**Review by teojunda — APPROVED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (kailashgautham) [reply docs/UserGuide.md:217]: I tried to modify it a bit to sound better - will be going with 

If an optional prefix is used without a value, it is assumed that you are creating a user without the value or deleting the already existing value.

Thanks for the advice!

File: src/main/resources/view/PersonListCard.fxml
Comment (kailashgautham) [reply src/main/resources/view/PersonListCard.fxml:1]: Ah so i dont think flowpane was doing anything actually... actually ngl neither of those are doing anything. ill remove it.

**Review by g-tejas — APPROVED** [review]
LGTM!

**Pull request merged**